doc/psy_layer.rst

```diff
@@ -108,10 +108,10 @@ API
 Schedule
 --------
 
-A PSy **Schedule** object consists of a tree of objects which can be
-used to describe the required schedule for a PSy layer
-subroutine. This subroutine is called by the Algorithm layer and
-itself calls one or more Kernels and/or implements any required
+A PSy **Schedule** object consists of a tree of objects (called Nodes
+in PSyclone) which can be used to describe the required schedule for a
+PSy layer subroutine. This subroutine is called by the Algorithm layer
+and itself calls one or more Kernels and/or implements any required
 Built-in operations. These objects can currently be a **Loop**, a
```

Comment:
s/These objects/The Node objects/ because of the intervening sentence describing the PSy layer subroutine.